modules/configuring-running-microshift-executable.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-running-microshift-executable"]
+= Configuring and running MicroShift
+:context: creating-microshift-development-environment
+
+Use the following procedure to configure and run a MicroShift deployment 
+
+.Prerequisites 
+
+* You have logged into your development virtual machine using your MicroShift user credentials. 
+* You have installed the `oc` CLI and `kubectl` utilities. 
+* You have updated CRI-O, networking, and file system customizations. 
+* You have set up your MicroShift development environment. 
+* You have configured and built MicroShift on your virtual machine. 
+* You have prepared the MicroShift executable runtime. 
```

Comment:
I'm not sure what this means. 

Users should just need to install MicroShift via the RPM we will publish.